### ISSUE TITLE
repr for event and list

### ISSUE THREAD

**Issue by Yikai-Liao** [Yikai-Liao/symusic#7]
For higher information density, I'm considering changing the repr of list of event ( like NoteTickList ), like this

When showing a list, I tend to hide they arguments' names, while the names are reserved for showing the single one
```text
Note(time=0, duration=118, pitch=69, velocity=117)
[Note(0, 118, 69, 117), Note(0, 118, 77, 62), Note(240, 238, 76, 62), Note(480, 238, 72, 62), Note(720, 238, 67, 62)]
```
And for track and score, I will turn to show the summary, following miditoolkit, like this:
```text
Score(ttype=Tick, tpq=480, begin=780, end=1431755, tracks=51, notes=60411, time_sig=97, key_sig=97, markers=97, lyrics=0)
Track(ttype=Tick, program=72, is_drum=false, name=PICCOLO, notes=1053)
```
And, when showing TrackList, the arguments' names won't be removed

@Natooz Do you have any suggestions?

**Comment by Yikai-Liao** [comment]
It seems hard to define customized `__repr__` for vectors. So I'll keep the issue.

**Comment by Yikai-Liao** [comment]
Implemented in `0.5.0b1`

**Comment by Yikai-Liao** [comment]
https://pypi.org/project/symusic/0.5.0/
